crates/ruff_linter/src/rules/flake8_simplify/rules/suppressible_exception.rs

```diff
@@ -115,7 +118,13 @@ pub(crate) fn suppressible_exception(
     };
 
     let exception = if handler_names.is_empty() {
-        "Exception".to_string()
+        if type_.is_none() {
+            // case where there are no handler names provided at all
+            "BaseException".to_string()
```

Comment:
This won’t work if `BaseException` is shadowed. Here is an example of to safely insert a builtin symbol:
https://github.com/astral-sh/ruff/blob/b35bf8ae073a47e12a98eea3eb4818d3695ff302/crates/ruff_linter/src/rules/flake8_bugbear/rules/unreliable_callable_check.rs#L78-L82